doc/network-zones.md

```diff
@@ -4,57 +4,149 @@ discourse: 12033,13128
+```
+
+The following examples show how to configure a zone for forward DNS records, one for IPv4 reverse DNS records and one for IPv6 reverse DNS records, respectively:
+
+```bash
+lxc network zone create lxd.example.net
+lxc network zone create 30.192.10.in-addr.arpa
+lxc network zone create 1.9.f.6.5.a.e.f.f.f.e.3.6.1.2.ip6.arpa
+```
+
+```{note}
+Zones must be globally unique, even across projects.
+If you get a creation error, it might be due to the zone already existing in another project.
+```
+
+You can either specify the configuration options when you create the network or configure them afterwards with the following command:
+
+```bash
+lxc network zone set <network_zone> <key>=<value>
+```
+
+Use the following command to edit a network zone in YAML format:
+
+```bash
+lxc network zone edit <network_zone>
+```
 
 ### Configuration options
 
-The following are network zone properties:
+The following configuration options are available for network zones:
 
-Property            | Type       | Required | Default | Description
+Key                 | Type       | Required | Default | Description
 :--                 | :--        | :--      | -       | :--
 peers.NAME.address  | string     | no       | -       | IP address of a DNS server
 peers.NAME.key      | string     | no       | -       | TSIG key for the server
-dns.nameservers     | string set | no       | -       | Comma separated list of DNS server FQDNs (for NS records)
+dns.nameservers     | string set | no       | -       | Comma-separated list of DNS server FQDNs (for NS records)
 network.nat         | bool       | no       | true    | Whether to generate records for NAT-ed subnets
-
-Additionally the `user.` key namespace is also supported for user-provided free-form key/value.
+user.*              | *          | no       | -       | User-provided free-form key/value pairs
 
 ## Add a network zone to a network
 
-This is controlled through `dns.zone.forward`, `dns.zone.reverse.ipv4` and `dns.zone.reverse.ipv6` in network configuration.
+To add a zone to a network, set the corresponding configuration option in the network configuration:
+
+- For forward DNS records: `dns.zone.forward`
+- For IPv4 reverse DNS records: `dns.zone.reverse.ipv4`
+- For IPv6 reverse DNS records: `dns.zone.reverse.ipv6`
+
+For example:
+```bash
+lxc network set <network_name> dns.zone.forward="lxd.example.net"
+```
 
 Zones belong to projects and are tied to the `networks` features of projects.
-It is possible to restrict projects to specific domains and sub-domains through the `restricted.networks.zones` project configuration key.
+You can restrict projects to specific domains and sub-domains through the `restricted.networks.zones` project configuration key.
 
 ## Add custom records
 
-It's possible to add additional records to a zone.
-On the CLI, this is done through `lxc network zone record`.
+A network zone automatically generates a DNS record for each instance.
```

Comment:
We should keep this consistent with the statement around automatic record generation above:

> LXD will then automatically manage forward and reverse records for all instances, network gateways and downstream network ports